src/app/mutation-page/mutation-page.component.html

```diff
@@ -1,48 +1,61 @@
-
-    <fieldset style="flex: 1 0 auto; display: flex; flex-direction: column; gap: 8px;">
-      <legend>Observable</legend>
-
-      <input id="observableTodo" placeholder="Input a new todo here..." [(ngModel)]="newTodo" style="margin-top: 16px;">
-      <button (click)="onAddTodo(newTodo)">Add Todo</button>
-      <button (click)="onAddTodoWithError(newTodo)">Add Todo with Error</button>
-      <button (click)="onResetMutation()">Reset Mutation</button>
-
-      <ng-container *ngIf="addTodo.result$ | async as addTodo">
-        <p *ngIf="addTodo.isPending">loading...</p>
-        <p *ngIf="addTodo.isIdle">Idling</p>
-        <div *ngIf="addTodo.isError" style="word-break: break-all;">
-          <p>Could not add the Todo</p>
+<div class="content">
+  <h1>Mutation Example</h1>
+
+  <p>This example shows how to use the Mutations API</p>
+  <p>To get a better understanding on what happens, open the <b>devtools</b> in the bottom right.</p>
+
+  <h4>Is Mutating: {{ addTodoMutationsActive() }}</h4>
+</div>
+
+<query-tabs>
+  <query-tab title="Signal API">
+    @if (addTodoSignalResult();as addTodo) {
+      <div class="block" style="margin: 4rem 0 2rem 0;">
+        <div class="field is-grouped is-horizontal">
+          <div class="control is-expanded">
+            <input class="input" placeholder="Todo title" id="signalTodo" type="text" [(ngModel)]="newTodo">
+          </div>
+          <div class="control">
+            <button
+              class="button is-primary-button"
+              (click)="onAddTodo(newTodo)"
+              [class.is-loading]="addTodo.isPending"
+              [disabled]="addTodo.isPending"
+            >
+              Add Todo
+            </button>
+          </div>
+          <div class="control">
+            <button class="button is-danger" (click)="onAddTodoWithError(newTodo)">Show Error</button>
+          </div>
+          <div class="control">
+            <button class="button is-black" (click)="onResetMutation()">Reset Mutation</button>
+          </div>
         </div>
-        <p *ngIf="addTodo.isSuccess">
-          {{ addTodo.data.title }}
-        </p>
-      </ng-container>
-    </fieldset>
-  </div>
-</div>
\ No newline at end of file
+      </div>
+      <div class="block content box">
+        <h4>Mutation Output</h4>
+        <code style="background-color: initial; max-height: 300px; overflow: hidden; overflow-y: auto; display: block">
+          <pre>{{ addTodo | json }}</pre>
+        </code>
+      </div>
+    }
+  </query-tab>
+  <query-tab title="Observable API">
+    <input id="observableTodo" placeholder="Input a new todo here..." [(ngModel)]="newTodo" style="margin-top: 16px;">
+    <button (click)="onAddTodo(newTodo)">Add Todo</button>
+    <button (click)="onAddTodoWithError(newTodo)">Add Todo with Error</button>
+    <button (click)="onResetMutation()">Reset Mutation</button>
+
+    @if (addTodo.result$ | async;as addTodo) {
+      <p *ngIf="addTodo.isPending">loading...</p>
```

Comment:
Let's use the new control flow syntax please

Reply:
Of course, this not finished yet though, just wanted to get your opinion on the layout so that i dont have to redo too much things.